web_company_color/models/res_company.py

```diff
@@ -74,34 +74,35 @@ def write(self, values):
                 "color_navbar_bg_hover",
                 "color_navbar_text",
             )
-            if "logo" in values:
-                if values["logo"]:
-                    _r, _g, _b = image_to_rgb(convert_to_image(values["logo"]))
-                    # Make color 10% darker
-                    _h, _l, _s = rgb_to_hls(_r, _g, _b)
-                    _l = max(0, _l - 0.1)
-                    _rd, _gd, _bd = hls_to_rgb(_h, _l, _s)
-                    # Calc. optimal text color (b/w)
-                    # Grayscale human vision perception (Rec. 709 values)
-                    _a = 1 - (0.2126 * _r + 0.7152 * _g + 0.0722 * _b)
-                    values.update(
-                        {
-                            "color_navbar_bg": n_rgb_to_hex(_r, _g, _b),
-                            "color_navbar_bg_hover": n_rgb_to_hex(_rd, _gd, _bd),
-                            "color_navbar_text": "#000" if _a < 0.5 else "#fff",
-                        }
-                    )
-                else:
-                    values.update(self.default_get(fields_to_check))
-
             result = super().write(values)
-
             if any([field in values for field in fields_to_check]):
```

Comment:
These 2 lines should be moved to the new method.

Reply:
I don't think that's the problem, locally is working well, btw the new method is launching a write with the values computed

Could be that the context has the key ignore_company_color?
https://github.com/OCA/web/pull/2157/files#diff-9b4dca5f4bdbc5450fc14eeba219185f4894189e94d026978af5a65841c5f85eR71

Reply:
Yes, you are right that it's called anyway, as after the write method is there.

Reply:
I don't see the problem right now.

Reply:
I test it on runboat and it's working

@Tardo, did you test it with the correct company?